OracleDatabase/RAC/OracleDNSServer/dockerfiles/latest/named.conf

```diff
@@ -0,0 +1,36 @@
+//
+// named.conf
+//
+// Provided by Red Hat bind package to configure the ISC BIND named(8) DNS
```

Comment:
I strongly encourage you to switch to using `dnsmasq` instead of `bind`. It's much smaller and far easier to configure.